backend/src/websockets/connection_manager.py

```diff
@@ -46,11 +46,12 @@ async def handle_message(self, ws: WebSocket, message: Message):
     async def broadcast(self, message: Message):
         for ws in self.websockets:
             if ws.application_state == WebSocketState.CONNECTED:
-                await ws.send_json(json.dumps({"type": message.type.value, "data": message.data}))
+                await ws.send_json({"type": message.type.value, "data": message.data})
```

Comment:
Do you think we should change from json.dumps to normal in other agents as well?